### PR TITLE
build: use Java/FX 19

### LINKED ISSUE #562: Update base Java/FX to 19
In order to prepare release of Scene Builder 19.0.0, the base JDK and JavaFX version needs to be updated to 19.

### PR DESCRIPTION
**Pull request by abhinayagarwal**
### Issue

Fixes #562

### Progress

- [x] Change must not contain extraneous whitespace
- [x] License header year is updated, if required
- [x] The PR name must follow the [pre-defined format](https://github.com/gluonhq/scenebuilder/blob/master/CONTRIBUTING.md)
- [x] Verify the contributor has signed [Gluon Individual Contributor License Agreement (CLA)](https://docs.google.com/forms/d/16aoFTmzs8lZTfiyrEm8YgMqMYaGQl0J8wA0VJE2LCCY)

### CODE REVIEW

**Comment by AlmasB** [comment]
After #565 all workflows should be happy yeah? Then we can review this after the other one is merged

**Comment by abhinayagarwal** [comment]
All checks have passed :)

**Review by AlmasB — APPROVED** [review]

File: .github/workflows/bundles-mac.yml [hunk 1]
```diff
@@ -10,11 +10,11 @@ on:
         required: true
         type: string
       java-version:
-        default: '18'
+        default: '19'
```
Comment (jperedadnr) [inline .github/workflows/bundles-mac.yml:13]: We also need to add a Mac M1 local Runner (possibly by copying this workflow)

Comment (abhinayagarwal) [reply .github/workflows/bundles-mac.yml:13]: best to tackle it in a different PR?

Comment (jperedadnr) [reply .github/workflows/bundles-mac.yml:13]: Sure, indeed

**Review by jperedadnr — APPROVED** [review]

**Pull request merged**